views/spots/configure.njk

```diff
@@ -0,0 +1,208 @@
+        </select>
+        <small>
+          {{ t('ui.spots.form.public.note') }}
+        </small>
+      {% endif %}
+
+      <details>
+        <summary>
+          {{ t('ui.spots.form.optionsForAdvancedUsers.label') }}
+        </summary>
+        <p>
+          {{ t('ui.spots.form.optionsForAdvancedUsers.note') }}
+        </p>
+        <label for="payload">
+          {{ t('ui.spots.form.optionsForAdvancedUsers.payload.label') }}
+        </label>
+        <textarea id="payload" type="text" name="payload" placeholder="{{ t('ui.spots.form.optionsForAdvancedUsers.payload.placeholder') }}">{{ spot.payload }}</textarea>
+        <small>
+          {{ t('ui.spots.form.optionsForAdvancedUsers.payload.note') }}
+        </small>
+      </details>
+
+    </fieldset>
+    <div id="success"></div>
+    <div id="errors"></div>
+    <button type="submit">
+      <i class="fas fa-save"></i>
+      Save
+    </button>
+  </form>
+{% endblock %}
+
+{% block javascript %}
+  <script>
+    const getLocationButton = document.getElementById("get-location-button");
+
+    getLocationButton.addEventListener("click", async () => {
+      if (!navigator.geolocation) {
+        return;
+      }
+
+      const position = await new Promise((resolve, reject) => {
+        navigator
+          .geolocation
+          .getCurrentPosition(resolve, reject);
+      });
+
+      const latitude = document.getElementById("latitude");
+      const longitude = document.getElementById("longitude");
+
+      latitude.value = position
+        .coords
+        .latitude
+        .toFixed(5);
+      longitude.value = position
+        .coords
+        .longitude
+        .toFixed(5);
+    });
+
+    const form = document.getElementById("form");
+    const success = document.getElementById("success");
+    const errors = document.getElementById("errors");
+
+    const latitude = document.getElementById("latitude");
+
+    form.addEventListener("submit", async (event) => {
+      event.preventDefault();
+
+      // Remove aria-invalid from all fields
+      document
+        .querySelectorAll('[aria-invalid]')
+        .forEach((elem) => {
+          elem.setAttribute('aria-invalid', 'false');
+        });
+
+      success.innerHTML = "";
+      errors.innerHTML = "";
+
+      // Get the form data
+      const toConfigure = document.getElementById("to-configure");
+      const id = document.getElementById("id");
+      const title = document.getElementById("title");
+      const description = document.getElementById("description");
+      const latitude = document.getElementById("latitude");
+      const longitude = document.getElementById("longitude");
+      const payload = document.getElementById("payload");
+      const redirection = document.getElementById("redirection");
+      const referenced = document.getElementById("referenced");
+      const configured = document.getElementById("configured");
+
+      const body = JSON.stringify({
+        title: title.value
```

Comment:
You can use the `||` operator (or `??` in other cases if you want to check more strictly that the value is `undefined` or `null`) to simplify these conditions a bit if needed:

`title: title.value || undefined`

Reply:
You're right!

Thank you, I'll update the code as soon as possible to reflect this. :)